package/MDAnalysis/core/universe.py

```diff
@@ -320,11 +331,18 @@ class Universe(object):
     .. versionchanged:: 2.5.0
         Added fudge_factor and lower_bound parameters for use with
         *guess_bonds*.
+
+    .. versionchanged:: 2.8.0
+        Added :meth:`~MDAnalysis.core.universe.Universe.guess_TopologyAttrs` API
+        guessing masses and atom types after topology
+        is read from a registered parser.
```

Comment:
list the new kwargs explicitly so that one knows which kwargs are actually available from 2.8.0 onwards (or add versionadded to the kwargs themselves)

Reply:
I'm sorry but how could I add versionadded to the kwargs? there are no specific kwargs at the moment for the API... it is to add flexibility to future implemented guessers

Reply:
@orbeckst I think all new kwargs added to `Universe.__init__` have been documented in the main docstring.

Reply:
ok, that's good then, thanks